genai/example_test.go

```diff
@@ -220,11 +262,12 @@ func ExampleGenerativeModel_GenerateContent_safetySetting() {
 			Threshold: genai.HarmBlockMediumAndAbove,
 		},
 	}
-	resp, err := model.GenerateContent(ctx, genai.Text("I want to be bad. Please help."))
+	resp, err := model.GenerateContent(ctx, genai.Text("I support Martians Soccer Club and I think Jupiterians Football Club sucks! Write a ironic phrase about them."))
```

Comment:
Does the word "sucks" actually appear in the other languages' samples? It seems inappropriate.

Reply:
It does indeed: https://github.com/google-gemini/generative-ai-python/blob/main/samples/safety_settings.py#L24

I suppose it's on purpose, to elicit a harm block from the model? The full prompt is very abstract, so it's probably not a big deal